src/index.js

```diff
@@ -1,8 +1,21 @@
 import pixelUrl from './pixelUrl'
 
-const img = document.createElement('img')
-img.src = pixelUrl(document)
-img.width = 1
-img.height = 1
-img.style = 'display:none;'
-document.body.appendChild(img)
+/**
+ * `factorialPixelDomain` can be setup by the host this way
+ * the cookie can be written in staging.factorialhr.com
+ * for example
+ */
+
+/* eslint-disable no-undef */
+const domain = window.factorialPixelDomain || 'https://factorialhr.com'
```

Comment:
why is this hardcoded? this forces you to do hacks with global variables :/

Reply:
What would you do? This package is meant to be used in a public page not as an npm package where you pass variables.

Reply:
Also it was fully hardcoded in previous versions. I didn't see another approach

Reply:
https://github.com/factorialco/factorial-pixel/pull/16